File: backend/src/main/java/com/surimap/offlinepackage/query/OfflinePackageInstallationStatus.java
package com.surimap.offlinepackage.query;

public record OfflinePackageInstallationStatus(
    String id,
    String incidentId,
    String policePhoneId,
    String policePhoneCode,
    String policePhoneName,
    String status,
    long version,
    long sequence,
    int manifestVersion,
    int activeManifestVersion,
    boolean readyForOfflineUse) {

  public OfflinePackageInstallationStatus(
      String id,
      String incidentId,
      String policePhoneId,
      String policePhoneCode,
      String policePhoneName,
      String status,
      long version,
      long sequence,
      int manifestVersion,
      boolean readyForOfflineUse) {
    this(
        id,
        incidentId,
        policePhoneId,
        policePhoneCode,
        policePhoneName,
        status,
        version,
        sequence,
        manifestVersion,
        manifestVersion,
        readyForOfflineUse);
  }
}

File: android/app/src/main/java/com/surimap/core/offline/OfflinePackageInstallationStatus.kt
package com.surimap.core.offline

data class OfflinePackageInstallationStatus(
    val incidentId: String,
    val policePhoneId: String,
    val manifestId: String,
    val manifestVersion: Int,
    val status: String,
    val totalItems: Int,
    val completedItems: Int,
    val failedItems: Int,
    val version: Long,
    val readyForOfflineUse: Boolean
)
